src/parsers/manifestjson.js

```diff
@@ -146,6 +150,17 @@ export default class ManifestJSONParser extends JSONParser {
     }
   }
 
+  validateIcons() {
+    const { icons } = this.parsedJSON;
+    Object.keys(icons).forEach((size) => {
+      const path = icons[size];
+      if (!this.io.files.hasOwnProperty(path)) {
+        this.collector.addError(messages.manifestIconMissing(path));
+        this.isValid = false;
+      }
+    });
+  }
```

Comment:
I actually wonder if we should verify that they're not empty and match their anticipated file-type (e.g a `.png` file is an actual PNG).